addons/packages/secretgen-controller/0.7.1/package.yaml

```diff
@@ -3,23 +3,41 @@ kind: Package
 metadata:
   name: secretgen-controller.community.tanzu.vmware.com.0.7.1
 spec:
+  valuesSchema:
+    openAPIv3:
+      type: object
+      additionalProperties: false
+      properties:
+        secretgenController:
+          type: object
+          additionalProperties: false
+          description: Configuration for secret gen controller
+          properties:
+            namespace:
+              type: string
+              default: secretgen-controller
+              description: Namespace for secret gen controller
+            createNamespace:
+              type: boolean
+              default: false
+              description: Whether to create namespace for secret gen controller if not present
   refName: secretgen-controller.community.tanzu.vmware.com
   version: 0.7.1
-  releaseNotes: "secretgen-controller 0.7.1 https://github.com/vmware-tanzu/carvel-secretgen-controller"
+  releaseNotes: secretgen-controller 0.7.1 https://github.com/vmware-tanzu/carvel-secretgen-controller
   licenses:
-    - "Apache 2.0"
+  - Apache 2.0
   template:
     spec:
       fetch:
-        - imgpkgBundle:
-            image: projects.registry.vmware.com/tce/secretgen-controller@sha256:4248e36490eb888d7f8bd0b62739a5acc3f178f67d8c2abfb3a6181b814c074e
```

Comment:
Question: If a maintainer uses the `package-template.yaml` above, will the SHA always be the one specified? If so, it is an error-prone process because maintainer could easily forget about updating it. That makes me think whether the package folder becomes cleaner if we only check in the changes from `schema.yaml` and `package.yaml` (that also helps remove some duplications). And provide a doc on how to add/update schema to package.

Reply:
Since we are adding new files like schema.yaml to the package , we need to run `make push-package` which generates new SHA for package.yaml. If the SHA is being updated in package.yaml, the question is person can forget to add SHA  in package-template.
I was thinking of adding a doc and adding some make commands to help in schema and package generation. What should be the location of doc?